Sources/Common/cmdArgs/impl/ListWindowsCmdArgs.swift

```diff
@@ -36,6 +39,8 @@ public struct ListWindowsCmdArgs: CmdArgs {
     fileprivate var all: Bool = false // ALIAS
 
     public var filteringOptions = FilteringOptions()
+    public var _sortOrder: SortOrder? = nil
+    public var sortOrder: SortOrder { _sortOrder ?? .treeOrder }
```

Comment:
```suggestion
    public var sortOrder: SortOrder = .treeOrder
```

The second variable seems redundant